science/proxmark3-iceman/Portfile

```diff
@@ -5,15 +5,16 @@ PortGroup           github 1.0
 PortGroup           qt5 1.0
 
 name                proxmark3-iceman
+version             4.14831
```

Comment:
no need to define `version` here, it will be deduced from your `github.setup` line